bundles/org.openhab.binding.kaleidescape/README.md

```diff
@@ -1,37 +1,32 @@
 # Kaleidescape Binding
 
-This binding is used to control and retrieve information from a Kaleidescape movie player.
-All movie player components including the original K-Player series, M Class Players, Cinema One, Alto, and Strato are supported.
-The 4 zone audio only KMUSIC-4000 is not supported at this time.
-As there are many good control options already available for these components, this binding focuses primarily on retrieving information
- for display purposes and to use in rules for controlling other Things such lighting, projector lens control, masking, etc.
-Basic playback transport controls are provided and any other command that is supported by the control protocol can be sent to the component through rules based commands.
-See [Kaleidescape-System-Control-Protocol-Reference-Manual.pdf](https://support.kaleidescape.com/article/Control-Protocol-Reference-Manual) for a reference of available commands.
-To simplify the design of the binding code, a different Thing instance is created for each component
- in a multi-zone system and each Thing maintains its own socket connection to the target component.
-Overall this binding supports the majority of information and commands available in the Kaleidescape control protocol but is by no means exhaustive.
-Any feedback or suggestions for improvement are welcome.
+The Ultimate Movie Player - Kaleidescape elevates every component in your theater with higher fidelity
+source material for the ultimate cinematic experience.
```

Comment:
This is copied from marketing material, is it really needed?

Reply:
I was trying to add a slight bit of excitement to otherwise dry documentation. What if I re-phrased it something like this:

> The Ultimate Movie Player - Kaleidescape elevates every component in your theater that now includes automatons via openHAB.

Reply:
I'll just take it out. I am having a hard time coming up with a slick opening sentence that hits all the right notes without too much marketing speak.